pkg/cmd/codespace/list.go

```diff
@@ -54,6 +54,7 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 	tp := utils.NewTablePrinter(a.io)
 	if tp.IsTTY() {
 		tp.AddField("NAME", nil, nil)
+		tp.AddField("DISPLAY NAME", nil, nil)
```

Comment:
I wonder if display name should be the first column, as the primary identifier we expect people to use?

Reply:
Is the display named usable as a primary identifier on the API level? My testing shows no:

```
$ gh api user/codespaces/improved%20succotash
gh: Not Found (HTTP 404)

$ gh api user/codespaces/mislav-github-github-q49929wx9
{
  "id": 2865300,
```

Reply:
No, but it it is how we expect people to identify them.